src/app/components/common/SelectField/MultiplySelect.tsx

```diff
@@ -24,19 +24,26 @@ const MultiplySelect = ({ fieldKey, placeholder }: { fieldKey: string, placehold
     };
 
     return (
-        <FormControl className={classes.SelectExtendClass} fullWidth>
+        <FormControl className={`${classes.SelectExtendClass} ${classes.multiplyInput}`} fullWidth>
             <Select
                 multiple
                 value={values}
                 onChange={handleChange}
-                placeholder={placeholder}
-                renderValue={(selected) => (
-                    <div >
+                displayEmpty
+                renderValue={(selected) => ( selected.length > 0 ?
```

Comment:
Better condition should be `selected?.length` without >0

Reply:
done